server/models/trip.js

```diff
@@ -12,8 +12,31 @@ const tripSchema = mongoose.Schema({
   },
   description: {
     type: String,
+    validate: [
+      {
+        validator: (rawValue) => {
+          try {
```

Comment:
The same as in front... It is not the most pleasant way to work with data, but in this case it is the least of evils...